storage/innobase/ut/ut0crc32.cc

```diff
@@ -342,11 +342,11 @@ allocations, would not hurt if called twice, but would be pointless. */
 void ut_crc32_init()
 {
 #ifndef HAVE_CRC32_VPMSUM
-# if defined(__GNUC__) && defined(HAVE_ARMV8_CRC) && defined(HAVE_ARMV8_CRYPTO)
-  if (crc32c_aarch64_available())
+# if defined(__GNUC__) && defined(HAVE_ARMV8_CRC)
+  if (const char *crc32c_implementation = crc32c_aarch64_available())
```

Comment:
A minor detail: in the common code formatting style, the preference is to not have a space before the assignment operator: `crc32c_implementation= crc32c_aarch64_available()`.